com.unity.render-pipelines.universal/Runtime/2D/Renderer2D.cs

```diff
@@ -29,14 +36,23 @@ public Renderer2D(Renderer2DData data) : base(data)
             m_Render2DLightingPass = new Render2DLightingPass(data);
             m_PostProcessPass = new PostProcessPass(RenderPassEvent.BeforeRenderingPostProcessing, data.postProcessData, m_BlitMaterial);
             m_FinalPostProcessPass = new PostProcessPass(RenderPassEvent.AfterRenderingPostProcessing, data.postProcessData, m_BlitMaterial);
-            m_FinalBlitPass = new FinalBlitPass(RenderPassEvent.AfterRendering, m_BlitMaterial);
+            m_FinalBlitPass = new FinalBlitPass(RenderPassEvent.AfterRendering + 1, m_BlitMaterial);
 
             m_UseDepthStencilBuffer = data.useDepthStencilBuffer;
 
-            m_AfterPostProcessColorHandle.Init("_AfterPostProcessTexture");
-            m_ColorGradingLutHandle.Init("_InternalGradingLut");
+            // We probably should declare these names in the base class,
+            // as they must be the same across all ScriptableRenderer types for camera stacking to work.
+            k_ColorTextureHandle.Init("_CameraColorTexture");
+            k_DepthTextureHandle.Init("_CameraDepthAttachment");
+            k_AfterPostProcessColorHandle.Init("_AfterPostProcessTexture");
+            k_ColorGradingLutHandle.Init("_InternalGradingLut");
 
             m_Renderer2DData = data;
+
+            supportedRenderingFeatures = new RenderingFeatures()
+            {
+                cameraStacking = true,
```

Comment:
Now that the 2D Renderer supports camera stacking, we should change the manual to reflect the new reality. @phi-lira Can the doc people on your side do this? Or should I ask our tech writer?

Reply:
Ideally your tech write but we can also help if needed.

Reply:
OK then I'll ask our tech writer to update the doc in another PR.